main.cpp

```diff
@@ -91,10 +91,10 @@ gardener.controlsPest();
 //  Noun: Surgeon 
 //  action 1: a surgeon performs surgical procedures
 surgeon.performsSurgeries();

```

Comment:
I'm marking this because:
1. your plain-english description is not an action, but is an ability or describes a state OR
2. the plain-english description is fine but the function name doesn't match OR
3. The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise